cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -29,7 +29,7 @@
 )
 from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory, LibraryFactory
 from xmodule.modulestore.tests.mongo_connection import MONGO_PORT_NUM, MONGO_HOST
-from xmodule.modulestore.tests.test_cross_modulestore_import_export import MongoContentstoreBuilder
+from xmodule.modulestore.tests.utils import MongoContentstoreBuilder
```

Comment:
Nit: note that there is an important statement from xmodule.modulestore.tests.utils directly below this (could combine all of them).